corehq/apps/users/util.py

```diff
@@ -126,18 +126,12 @@ def user_id_to_username(user_id, use_name_if_available=False):
         return raw_username(user_object['username']) if "username" in user_object else None
 
 
+@quickcache(['user_id'])
 def cached_user_id_to_username(user_id):
-    if not user_id:
+    if not user_id or user_id == 'system':
```

Comment:
nit: this should probably be `SYSTEM_USER_ID`

Reply:
Nice @orangejenny and @esoergel. When looking at that I discovered `WEIRD_USER_IDS` which is a convenient list of all such nonsense IDs that don't warrent a DB lookup, and decided to just use that

Reply:
Nice. That's also an excellent name for a constant.

Reply:
> nonsense IDs

being harsh on the non-existent system user Danny.